### DISCUSSION TITLE
r2modmanPlus on steamdeck

### DISCUSSION THREAD

**Discussion by ddt3** [ebkr/r2modmanPlus#966] [Q&A]
I have r2modmanPlus running on my (Manjaro) desktop system. I use r2modmanPlus to start Valheim with a specific "mod profile" and that works really great. 
I also have a Steam deck (running something quite similar to Manjaro) and  I can get the Valheim mods to work but only if:
- I copy the mods that have been downloaded by r2modmanPlus to the Valheim directory. 
- The start-up setting for valheim in Steam is: `./start_bepinex_game.sh || %command%`

I can't get it to work when starting from r2modmanPlus (using the correct locations and r2modmanplus proposed start-up command). When I tested it on the command line I got a message about "content missing" (or something similar).

note that I use the native linux version of the game in both cases...

Is there anyone who uses r2modmanplus to start valheim on the steam deck?
(note: I am not asking about "using valheim modded" that works as described above)

**Comment by MythicManiac** [comment]
https://github.com/ebkr/r2modmanPlus/wiki/Modding-on-a-Steam-Deck-and-Linux check this wiki page.

That wiki is slightly outdated too, you should now be able to replace skip the config editing just by dropping an empty `.forceproton` file at the game install directory.

**Reply by ddt3** [reply]
I did read that page but dismissed it because the introduction (and the explanations) talk about use of proton and how the game is errornously seen as a native Linux game. But valheim is a native Linux game..

So I do not quite understand how .forceproton would help here, I run valheim natively, not through proton. And modding works fine on both Linux systems (desktop and steamdeck),it's just that on steamdeck  it appears you cannot starts it from another folder,I have to copy the mods to the Valheim foldee
